src/link.js

```diff
@@ -36,8 +36,11 @@ const linkDependencyAndroid = (androidProject, dependency) => {
     log.info(`Android module ${dependency.name} is already linked`);
     return null;
   }
+  
+  // Legacy support for top-level params that is Android-only
```

Comment:
Because some of the developers already defined `params` top-level. Since they are currently Android-only (more specifically, applied to Android only because iOS is not yet made), I assume they wanted it for Android. It's a legacy so that works as is after we move to platform-specific settings and can be removed later after all of them upgrade to `config.android.params`